### PR TITLE
Update vscode.mdx

### PR DESCRIPTION
**Pull request by visualstuart**
Add step-by-step instructions for enabling the Legacy Code Lens in VSCode.

### CODE REVIEW

File: docs/usage/vscode.mdx [hunk 1]
```diff
@@ -15,6 +15,9 @@ In the latest [PowerShell](https://marketplace.visualstudio.com/items?itemName=m
 
 Actually there is a bug, and the option is called Enable Legacy Code Lens, and is enabled by default and should be disabled for Pester 5. 😁 Take advantage of this and go try it right now!
 
+**Instructions (for VSCode on Windows)** In Visual Studio Code, select **File | Preferences | Settings** (or type **Ctrl+,**). In the **Settings** window, under the **User** tab, select **Extensions | PowerShell Configuration**. Uncheck the **Pester: Use Legacy Code Lens** checkbox. Close the **Settings** window.
```
Comment (fflaten) [inline docs/usage/vscode.mdx:18]: Generalize to open Settings -> ... to avoid being Windows/only?

Comment (visualstuart) [reply docs/usage/vscode.mdx:18]: Sure, but I only have a Windows OS, so I don't know what the experience is elsewhere. How to proceed?

Comment (fflaten) [reply docs/usage/vscode.mdx:18]: AFAIK, VSCode is mostly identical cross-platform besides maybe the menu bar (+ ctrl/cmd) on Mac. So I'd just skip the first sentence and start directly with "In the Settings window...".

Comment (nohwnd) [inline docs/usage/vscode.mdx:18]:
```suggestion
**Instructions:** In Visual Studio Code, select **File | Preferences | Settings** (or type **Ctrl+,**). In the **Settings** window, under the **User** tab, select **Extensions | PowerShell Configuration**. Uncheck the **Pester: Use Legacy Code Lens** checkbox. Close the **Settings** window.
```

**Comment by fflaten** [comment]
Thanks for your contribution and sorry for the delay! 🙌

_Btw. this page will be rewritten soon, but I wanted to make sure this got included properly first_

**Pull request merged**